src/App.css

```diff
@@ -61,6 +61,44 @@
    vertical-align: middle;
 }
 
+.delete-button {
+  margin-left: 10px;
+  width:  48px;
+  height: 48px; 
+  display: flex;
+  border-radius: 10px;
+}
+
+.new-task-container{
+  margin-top: 50px;
+  display: flex;
+  width: 770px;
+  height: 67px;
+  text-align: left;
+}
+
+
+.newtask-name {
+  width: 730px;
+  height: 48px;
+  text-align: left;
+  border-radius: 10px;
+  display: flex;
+}
+
+.newtask-button {
+  margin-left: 10px;
+  padding: 15px 0px 0px 25px;
+  display: flex;
+  width:  107.97px;
+  height: 48px; 
+  border: 1px solid;
+  border-radius: 10px;
+  background-color: rgb(241, 106, 241);
+  font-size: 6;
```

Comment:
I think the font-size is `16px` instead of just `6`. Also, always add the unit (`px`, `rem`, etc) unless the value is 0